src/main/config/messages_template.xml

```diff
@@ -7,413 +7,415 @@ Copyright 2005, 2006 IBM Corporation
-  
+
   <message id="DOTJ026E" type="ERROR">
-    <reason>The "topic merge" did not generate any output.</reason>
-    <response>Correct any earlier transform errors and try the build again, or see the DITA-OT documentation for additional causes.</response>
+    <reason>The 'topic merge' process did not generate any output.</reason>
+    <response>Correct any earlier errors and try the build again, or see the documentation for additional causes.</response>
   </message>
 
   <message id="DOTJ028E" type="ERROR">
-    <reason>No format attribute was found on a reference to file '%1', which does not appear to be a DITA file.</reason>
-    <response>If this is not a DITA file, set the format attribute to an appropriate value, otherwise set the format attribute to "dita".</response>
+    <reason>No @format attribute was found on a reference to the '%1' resource, which does not appear to be DITA.</reason>
+    <response>If this is not a DITA resource, set the @format attribute to an appropriate value; otherwise set the @format attribute to 'dita'.</response>
   </message>
 
   <!-- Deprecated since 3.5 -->
   <message id="DOTJ029I" type="INFO">
-    <reason>No 'domains' attribute was found for element '&lt;%1&gt;'.</reason>
-    <response>This generally indicates that your DTD or Schema was not developed properly according to the DITA specification.</response>
-  </message> 
-  
+    <reason>No @domains attribute was found for the &lt;%1&gt; element.</reason>
+    <response>This generally indicates that the DTD or schema was not developed properly according to the DITA specification.</response>
+  </message>
+
   <message id="DOTJ030I" type="INFO">
-    <reason>No 'class' attribute for was found for element '&lt;%1&gt;'.</reason>
-    <response>The element will be processed as an unknown or non-DITA element.</response>
-  </message> 
-  
+    <reason>No @class attribute was found for the &lt;%1&gt; element.</reason>
+    <response>Processing as an unknown or non-DITA element.</response>
+  </message>
+
   <message id="DOTJ031I" type="INFO">
-    <reason>No specified rule for '%1' was found in the ditaval file. This value will use the default action, or a parent prop action if specified.</reason>
-    <response>To remove this message, you can specify a rule for '%1' in the ditaval file.</response>
-  </message> 
-  
-  <!-- 2012-06-12: Still need to reproduce and edit this message -->
+    <reason>No rule for '%1' was found in the DITAVAL file. Using the default action, or a parent prop action if specified.</reason>
+    <response>To remove this message, specify a rule for '%1' in the DITAVAL file.</response>
+  </message>
+
   <message id="DOTJ033E" type="ERROR">
-    <reason>No valid content is found in topicref '%1' during chunk processing.</reason>
-    <response>Please specify an existing and valid topic for the topicref.</response>
-  </message> 
-  
+    <reason>No valid content found in topicref '%1' during chunk processing.</reason>
```

Comment:
Nit, that is not new to this message -- the "valid content" is not technically "in the topicref X", it is really "no valid content in the referenced resource X". Not a blocker for merging this, noting in case it's worth fixing.